atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -82,6 +83,9 @@ def run(self):
                 self._base_image_build = self.wait_for_parent_image_build(nvr)
 
         for img, local_tag in self.workflow.builder.parent_images.items():
+            if base_image_is_custom(img.to_str()):
```

Comment:
Since add_filesystem runs before koji_parent, isn't this a null-op?  Or is it possible to have an non-custom base image and a custom base image for a parent?

Reply:
yes we can have for example
FROM koji/image-builder
FROM fedora